app/controllers/hyrax/paged_resources_controller.rb

```diff
@@ -16,5 +16,27 @@ class PagedResourcesController < ApplicationController
 
     # Use this line if you want to use a custom presenter
     self.show_presenter = Hyrax::PagedResourcePresenter
+
+    def pdf
+      # PagedResourcePDF.new(presenter, quality: params[:pdf_quality]).render(pdf_hyrax_paged_resource_path)
+      # redirect_to main_app.download_path(presenter)
+
+      resource = PagedResource.find(params[:id])
+      # TODO: if tmp/pdfs/ doesn't exist, mkdir it
```

Comment:
you could try:

```FileUtils.mkdir_p(Rails.root.join('tmp', 'pdfs'))```

Reply:
Pertaining to another story, @dlpierce indicated that for our enterprise Docker context at IU, using the system temp dir may be preferable, as ImportUrlJob does:
https://github.com/samvera/hyrax/blob/master/app/jobs/import_url_job.rb#L71-L85